geopyspark-geotrellis/src/main/scala/geopyspark/geotrellis/TemporalTiledRasterLayer.scala

```diff
@@ -277,6 +277,14 @@ class TemporalTiledRasterLayer(
     TemporalTiledRasterLayer(zoom, tileLayer)
   }
 
+  def addTemporal(temporalKey:TemporalKey): TiledRasterLayer[SpaceTimeKey] = {
+    val bounds = rdd.metadata.bounds.get
```

Comment:
I'm not sure if we need this? The cube already has a temporal dimension in this case, so why would we add one again?